src/server/claimQueue/claimQueueAPI.js

```diff
@@ -9,19 +9,38 @@ import { ClaimQueueProps } from '../db/mongo/models/props'
 
 import conf from '../server.config'
 import { wrapAsync } from '../utils/helpers'
+import logger from '../../imports/logger'
+
+const defaultLogger = logger.child({ from: 'ClaimQueue ' })
 
 const ClaimQueue = {
-  async setWhitelisted(user, storage, log) {
-    //if user has passed, then we mark that in claim queue and tag the user
-    return Promise.all([
```

Comment:
why was promise all removed?

Reply:
@sirpy i've commented the reason in the code below:

> // Mautic calls could took a lot of time and cause ZoOm timeout
>  // so let's do it 'in background'
>  // also we won't call mautic if user.mauticId is empty or null